packages/common/src/util.js

```diff
@@ -30,6 +30,7 @@ var is = require('is');
 var request = require('request').defaults({
   timeout: 60000,
   gzip: true,
+	forever: true,
```

Comment:
I think this is a tab, could you replace it with two spaces? Sorry to be *that* guy :(